README.md

```diff
@@ -34,6 +34,8 @@ and running the following command:
       ```
       make install_conda
       ```
+      Optionally, if you work with multi-GPU environment and want to have Nvidia's collective communication calls
```

Comment:
I would remove the spaces at the start so this goes in the same line as the `To installl...` at the top.

`want to have` -> `want to use`

Also I would try not to refer to the documentation and provide some of the basic details here. In general, one would always expect the documentation to be more accurate but as a simple quick start we could say:

Install your environment with:
```
make install_conda_nccl
```